go-selinux/rchcon.go

```diff
@@ -12,7 +12,22 @@ import (
 )
 
 func rchcon(fpath, label string) error {
+	slowMode := false
+	// If the current label matches the new label, assume
+	// other labels are correct.
+	if currentLabel, err := lFileLabel(fpath); err == nil {
+		if label == currentLabel {
+			slowMode = true
+		}
+	}
```

Comment:
Slightly wondering if it's easier to read if it's a one-liner;

```suggestion
	if currentLabel, err := lFileLabel(fpath); err == nil && label == currentLabel {
		slowMode = true
	}
```